resources.tf

```diff
@@ -247,6 +461,27 @@ resource "aws_security_group_rule" "analytical_dataset_generation_ingress" {
   security_group_id = aws_security_group.analytical_dataset_generation.id
 }
 
+resource "aws_security_group_rule" "egress_https_to_vpc_endpoints" {
+  description              = "egress_https_to_vpc_endpoints"
+  from_port                = 443
+  protocol                 = "tcp"
+  security_group_id        = aws_security_group.analytical_dataset_generation.id
+  to_port                  = 443
+  type                     = "egress"
+  source_security_group_id = data.terraform_remote_state.internal_compute.outputs.vpc.interface_vpce_sg_id
+
+}
+
+resource "aws_security_group_rule" "ingress_https_vpc_endpoints_from_emr" {
```

Comment:
I think this needs to be done in the vpc enpoint to accept the connection from our SG. @mattburgess ?

Reply:
As this emr code exists in a different repo which is aws-analytical-dataset-generation than in aws-internal-compute repo where vpc code is . we will have to this way as an alternative